packages/@aws-cdk/aws-autoscaling-api/package.json

```diff
@@ -20,7 +20,11 @@
         "signAssembly": true,
         "assemblyOriginatorKeyFile": "../../key.snk"
       },
-      "sphinx": {}
+      "sphinx": {},
+      "python": {
+        "distName": "aws-cdk.autoscaling-api",
```

Comment:
I think these should be aws-autoscaling

For the same reasons we added the prefix in npm